filter/etc/e-smith/templates/etc/rspamd/rspamd.local.lua/10base

```diff
@@ -0,0 +1,15 @@
+{
+    if ( $rspamd{'SpamCheckStatus'} eq 'enabled') {
+        $OUT .= "\n -- symbols/lua rules are not disabled\n";
```

Comment:
My brain is confused by negative logic :D

```suggestion
        $OUT .= "\n -- all symbols/lua rules are enabled\n";
```

Reply:
done